var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAwkward(PythonPackage):
+    """ROOT I/O in pure Python and NumPy."""
```

Comment:
It looks like this description is from the uproot package, not awkward.

Reply:
Fixed